autobuild/workflow.md

```diff
@@ -0,0 +1,218 @@
+Not directly applicable to 3rd party developers who are simply looking to list a coin because the workflow includes building a Docker image in the blocknetdx Docker hub repo. 3rd party developers could choose to branch the https://github.com/blocknetdx/dockerimages repo and adapt it to push the image to their own Docker hub repo, or reach out to Blocknet team members for assistance through [Discord #block-dx-listing](https://discord.gg/WnaygWwqFy) or a Github issue. 
+
+* Go to https://github.com/blocknetdx/dockerimages/actions and choose the BUILD IMAGE FROM TEMPLATE workflow. 
+* Click the *Run workflow* dropdown.
+* Choose the appropriate branch (most likely master) in the *Use workflow from* dropdown.
+* Enter the URL of your branch in the *Full path of a branch where the config can be found.* field, eg: https://raw.githubusercontent.com/walkjivefly/blockchain-configuration-files/bump-BLOCK-v4.3.3
+* Enter the full name of the coin you wish to create a Docker image for in the *The full name of a wallet. Example - bitcoin, dash, blocknet.* field
+* Enter the version you wish to build in *Tag or version of an image, please check manifest-latest.json. By default it takes the newest version.*
+* Click the green *Run workflow* button.
+
+If successful, the workflow takes around 30 minutes to build and push a new image to Docker hub. The image will have a -staging tag at this point signifying it has not been tested and released for public use.
+
+## Build exrproxy-env deployment package
+**Audience:**
+Not applicable to 3rd party developers who are simply looking to list a coin.
+
+* These steps are based closely on https://docs.blocknet.co/service-nodes/setup/#prepare-to-deploy-service-node.
+* Clone the https://github.com/blocknetdx/exrproxy-env repo to your local machine. You need to create a deploy script for the servicenode which will oversee your test trades, and a deploy script for the two trading nodes.
+* cd exrproxy-env/autobuild
+* cp examples/alldaemons.yaml custom.yaml
+* Edit custom.yaml to create a (testnet) servicenode which includes the coin to be tested and any counter-party coin you want to use.
+* Run app.py specifying the path for your branch pushed at step 8 above, eg:
+```
+python3 app.py --branchpath https://raw.githubusercontent.com/walkjivefly/blockchain-configuration-files/bump-thing-v4.2.2
+```
+* Copy the newly created dockercompose-custom.yaml to your servicenode machine.
+* Edit custom.yaml to create a (testnet) trading node.
+* Run app.py again, as above, to create the dockercompose-custom.yaml to be used on the trading machines.
+* Copy the trading dockercompose-custom.yaml to the trading machines. 
+
+## Testing (CLI)
+**Audience:**
+Not directly applicable to 3rd party developers, although they could choose to follow a similar process if they don't want to jump through the hoops required (at this time) to test with BlockDX.
+
+* Bring up the Docker containers on all 3 machines and let them sync if necessary.
```

Comment:
Minor request:
`Bring up the Docker containers on all 3 machines and let them sync if necessary.`
Change to:
`Bring up the Docker containers on all 3 machines mentioned above and let them sync if necessary.`